static/javascripts/app.js

```diff
@@ -277,6 +320,7 @@ function playNote(buffer, noteTime, startTime, endTime, gainNode) {
 
   voice.connect(gainNode)
   gainNode.connect(currentLastNode);
+  gainNode.connect(recordingDest);
```

Comment:
Is it ok to always connect the node here?
Or should we just connect if we are recording?

Reply:
I think it allows to get sounding samples that have started before hitting the record button, which is nice